### PR TITLE
feat: Add API doc

### PR DESCRIPTION
**Pull request by gaocegege**
The API doc will be in http://localhost:8080/swagger/index.html#/

image

Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -159,6 +163,9 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+swag: swag-install
```
Comment (kemingy) [inline Makefile:166]: I think this one also requires `make build-local`.
```suggestion
swag: swag-install build-local
```

**Review by kemingy — COMMENTED** [review]
Please `rm pkg/docs/*` and add them to `.gitignore`

File: Makefile (hunk 1 above)
Comment (gaocegege) [reply Makefile:166]: It does not require a fresh build.

Comment (kemingy) [reply Makefile:166]: I see.

**Comment by gaocegege** [comment]
PTAL @kemingy 

Comments addressed.

**Review by kemingy — APPROVED** [review]
/lgtm

**Comment by gaocegege** [comment]
@kemingy 

Sorry, I forgot to configure the permissions. It should work now.

**Comment by kemingy** [comment]
/lgtm

Let me test.

**Pull request merged**